shared/lib-camel-connector/src/main/java/gov/va/vro/camel/RabbitMqCamelUtils.java

```diff
@@ -39,6 +43,11 @@ public static RouteDefinition fromRabbitmq(
     return fromRabbitmq(builder, rabbitmqConsumerEndpoint(exchangeName, routingKey));
   }
 
+  public static RouteDefinition fromRabbitmqFanoutExchange(
+      RouteBuilder builder, String fanoutExchange, String routingKey) {
```

Comment:
The params being on the next line makes it hard to read (for me).  It took me like 30 seconds to figure out what's going on here.  Which is alot for a 1-line function.

Reply:
SpotlessJava is forcing this format. The line is longer than it expects so it moves the parameters on their own line.

Reply:
Gocha.